src/app/find-help/transport-search/transport-search-form/transport-search-form.component.ts

```diff
@@ -47,7 +52,8 @@ export class TransportSearchFormComponent implements OnInit, OnDestroy {
     }
 
     this.formChangesSubscription = this.ngForm.form.valueChanges.subscribe((form) => {
-      this.showClearBtn = Object.values(form).some((el) => el !== undefined);
+      this.formCopy = Object.assign({}, form);
```

Comment:
ja wolę object literal
`this.formCopy = { ...form };`
ale co kto woli i oby było konsekwentnie (nie wiem jak mamy w aplikacji) :)

Reply:
Object.assign nie było nigdzie używane, spread operator raz, dla mnie bez różnicy, ale rzeczywiście ważne, żeby było spójnie. Poprawione.